packages/material-ui-lab/src/AvatarGroup/AvatarGroup.test.js

```diff
@@ -47,4 +47,54 @@ describe('<AvatarGroup />', () => {
     expect(container.querySelectorAll('img').length).to.equal(2);
     expect(container.textContent).to.equal('+2');
   });
+
+  it('should display all avatars with default (circular) variant', () => {
+    const { container } = render(
+      <AvatarGroup max={3}>
+        <Avatar src="image-url" />
+        <Avatar src="image-url" />
+        <Avatar src="image-url" />
+        <Avatar src="image-url" />
+      </AvatarGroup>,
+    );
+    expect(container.querySelectorAll('.MuiAvatar-root').length).to.equal(
```

Comment:
These assertions should be inverted. You should query by test id and assert that it has or doesn't have a specific class. Otherwise the error message is unhelpful once this test fails ("expected {} to equal null")

Reply:
hi! PR updated~ what do you think about the revised test cases?